### ISSUE TITLE
`vpx: all the codecs have failed! (vp8, vp9)`

### ISSUE THREAD

**Issue by stdedos** [Xpra-org/xpra#3389]
I am sorry to bring out issues one at a time, buuut, I just noticed ...

```
     30 2021-12-16 18:42:46,235 Handshake complete; enabling connection
     31 2021-12-16 18:42:46,285 vp8 codec encoding error 8: b'Invalid parameter'
     32 2021-12-16 18:42:46,285 vpx: vp8 encoding failed: data is empty and there are no delayed frames!
     33 2021-12-16 18:42:46,285 vpx: vp9 encoding failed: 'vp9'
     34 2021-12-16 18:42:46,285 vpx: all the codecs have failed! (vp8, vp9)
```

If there was a consistent way that loglevels started with ... 😕 
I am considering "improving" my pager (https://github.com/sharkdp/bat) sometime that any line that contains `(error|fail)` would be fully highlighted in red ...

1. ```
    Server: Ubuntu 20.04.3 LTS / 4.3-r30682 (81ce25817)
    /usr/bin/xpra
    shadow
    :0
    --title=@title@ on @@/@server-display@
    --ssh=plink -ssh -agent
    --desktop-scaling=1.25
    --speaker=off
    --exit-with-client=yes
    --splash=no
    --start-child=systemd-inhibit --what=handle-lid-switch sleep 1d
    --env=XPRA_PROXY_START_UUID=99992f3b042c49c5991d51d63921b382
    --daemon=yes
    --systemd-run=no
   ```
2. `"Xpra-Python3-x86_64_4.3-r30655\xpra_cmd" shadow ssh://user@ip/0 --ssh="plink -ssh -agent" --modal-windows=no --desktop-scaling=1.25 --headerbar=off --title="@title@ on @@/@server-display@" --start-child="systemd-inhibit --what=handle-lid-switch sleep 1d" --start-on-last-client-exit="systemd-inhibit --what=handle-lid-switch sleep 5m" --microphone=off --speaker=off --webcam=no --pulseaudio=no`
3. specific action to trigger the bug

**System Information (please complete the following information):**
- Server OS: Ubuntu 20.04.3 LTS
- Client OS: Microsoft Windows [Version 10.0.18363.836]
- Xpra Server Version (see above)
- Xpra Client Version (see above)

**Comment by totaam** [comment]
I had seen this error but then it went away.... and now it's back. Will fix.
In most cases, we have alternatives (ie: `x264`) so this is not a huge problem if `vp8` and `vp9` get disabled during the self tests.

---

> If there was a consistent way that loglevels started with ... confused

Any error will show up highlighted in red in the output, just not when it's sent to a log file. Use `--no-daemon` to see it.

**Comment by totaam** [comment]
Breakage: d92e91a5bb466bcab6208905561b031d51e9a236
Fix: 51f77a3b4d074d8c93c91b7b596141cf0f88003b
